docker-compose.yml

```diff
@@ -10,6 +10,7 @@ services:
       - PORT=3000
       - MONGODB_URI=mongodb://mongodb-node:27017/anythink-market
       - GITHUB_CODESPACES_PORT_FORWARDING_DOMAIN=${GITHUB_CODESPACES_PORT_FORWARDING_DOMAIN}
+      - OPENAI_API_KEY="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"
```

Comment:
It is very bad idea to store your keys in repository. You can do like this only locally for testing, but not pushing it to codebase.